### PR TITLE
Added Procfile, updated Database URI

### PR DESCRIPTION
**Pull request by enigmaeth**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "query-server",
+    "fossasia"
+  ],
+  "env": {
+    "APP_SECRET_TOKEN": {
+      "generator": "secret"
+    },
+    "ON_HEROKU": "true",
+    "FORCE_SSL": "true",
+    "INVITATION_CODE": {
```
Comment (niranjan94) [inline app.json]: These are all not needed for query-server

Comment (enigmaeth) [inline app.json]: Oh, I was just testing! Will remove them.

File: app.js [hunk 2]
```diff
@@ -10,13 +10,14 @@ var ejs = require('ejs');
 var favicon = require('serve-favicon');
 var spawn = require('child_process').spawn;
 
-var url = 'mongodb://localhost:27017/query-server';
+//var url = 'mongodb://localhost:27017/query-server';
+var url = 'mongodb://user:pass@host:port/db'
```
Comment (niranjan94) [inline app.js]: Hardcoding wont work. You'll have to read it from the ENV Variable set by Heroku

Comment (enigmaeth) [inline app.js]: Okay. I will fix that too.

**Pull request merged**